roxctl/deployment/check/check.go

```diff
@@ -172,6 +174,12 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
+	if d.cluster != "" && d.namespace == "default" {
+		d.env.Logger().WarnfLn("Warning: The namespace defaulted to \"default\", if you want to check deployments in a custom namespace set it with '--namespace [NAMESPACE]' or '-n [NAMESPACE]'\n")
+	}
+	if d.cluster == "" && d.namespace != "default" {
+		d.env.Logger().WarnfLn("Warning: Namespace has been set, but cluster remains unset. Setting the namespace only makes sense when also setting the cluster with '--cluster [CLUSTER]'.\n")
```

Comment:
You can omit the prefixed `Warning:`
```suggestion
		d.env.Logger().WarnfLn("Namespace has been set, but cluster remains unset. Setting the namespace only makes sense when also setting the cluster with '--cluster [CLUSTER]'.\n")
```

Reply:
Done